system-test/test-controller.ts

```diff
@@ -85,7 +86,7 @@ describe('Controller', function () {
 
   it('should pass success on timeout', done => {
```

Comment:
Agreed!  The challenge that I have at the moment is finding a suitable mock for the database.  nock doesn't do the trick since the client isn't using the rest api.  Any suggestions?